Documentation/Guided-Tour/file-browser.lisp

```diff
@@ -1,25 +1,21 @@
-(eval-when (:compile-toplevel)
-  (asdf:oos 'asdf:load-op :clim)
-  (asdf:oos 'asdf:load-op :clim-clx))
+(cl:eval-when (:compile-toplevel :load-toplevel :execute)
+  (cl:map cl:nil #'asdf:load-system '(:mcclim :cl-fad)))
```

Comment:
I thing using `ql:quickload` will be better everywhere. It will be simpler, much less chance of failing, and wouldn't output tons of warnings.

Reply:
I can do that.